grafana-plugin/src/components/MatchMediaTooltip/MatchMediaTooltip.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC, useEffect, useState } from 'react';
+
+import { Tooltip } from '@grafana/ui';
+import { debounce } from 'throttle-debounce';
+
+interface MatchMediaTooltipProps {
```

Comment:
you can pick props from `grafana/ui` `TooltipProps` like
` type MatchMediaTooltipProps = Pick<TooltipProps, 'placement'|'content'> `

Reply:
I'm not sure if you can extend that and add new fields, plus Grafana doesn't seem to import interfaces for props.

Reply:
yep it's possible `type MatchMediaTooltipProps = Pick<TooltipProps, 'placement'|'content'>& {title: string}`, sometimes Grafana exports, sometimes not,  so let's skip this  if Grafana doesn't export in this case